docs/Audit/AuditDiffer.md

```diff
@@ -0,0 +1,57 @@
+# AuditDiffer
+
+The `AuditDiffer` is an object differ, specifically built for generating diffs that can go into an audit log.
+
+In list form it...
+
+- will find changes made to `public` fields and properties
+- allows you to exclude certain fields or properties using the `AuditAttribute`
+- allows you to "mask" certain fields or properties using the `AuditAttribute` to set an [IAuditValueMasker](../ValueMasking)
+
+It is built on top of the context agnostic [RapidCore.Diffing.StateChangeFinder](../../Diffing/StateChangeFinder).
+
+Note that it uses `IRapidContainerAdapter` to create `IAuditValueMasker` instances.
+
+```csharp
+using RapidCore.Audit;
+
+class SomeThing
+{
+    public string Name { get; set; }
+
+    [Audit(ValueMasker=typeof(PasswordAuditValueMasker))]
+    public string Password { get; set; }
+
+    [Audit(Include=false)]
+    public string NotReallyRelevantToTheLog { get; set; } = "hephey";
+}
+
+class SomeController
+{
+    private readonly AuditDiffer auditDiffer;
+
+    public void AaaaaandAction()
+    {
+        var oldState = new SomeThing { Name = "T-Rex", Password = "NunYa" };
+        var newState = new SomeThing { Name = "Generic pop star", Password = "WhoGivesAPoop" };
```

Comment:
Set the `NotReallyRelevantToTheLog` to two different values to show that the output remains the same, just for the examples sake.

Reply:
True